server/jobs/deleteUnverifiedAccounts/deleteUnverifiedAccounts.js

```diff
@@ -0,0 +1,17 @@
+const { pool } = require('../../models/database')
+
+const deleteUnverifiedAccounts = () => {
+  pool.query(
+    `DELETE FROM users WHERE verified != 1 AND NOW() >= otp_valid_upto`,
```

Comment:
```suggestion
    `DELETE FROM users WHERE verified IS NULL AND NOW() >= otp_valid_upto`,
```
Default value of verified is NULL, so we need to check this way, because if you compare value with `=` or `!=`, MySQL only checks the rows with non-null columns. It ignores NULL values.